### PR TITLE
AFM: Note about the timing issue in the UI

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

**Review by nbattiato — APPROVED** [review]

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can create a monitor by filling in fields if you do not know how to write tests in the Behavior Driven Development (BDD) test-writing language or how to use the BAT CLI. This method of creating a monitor creates a test suite that tests one or more endpoints, and schedules the test suite to run at intervals that you select.
+You can create a monitor by filling in fields if you do not know how to write tests in the Behavior Driven Development (BDD) test-writing language or how to use the BAT CLI. This method of creating a monitor creates a test that tests one or more endpoints, and schedules the test suite to run at intervals that you specify.
```
Comment (harshakirank) [inline modules/ROOT/pages/afm-create-monitor.adoc:6]: "write tests in the Behavior Driven Development (BDD) test-writing language" - This is confusing, did we mean if a user wants to create simple ping tests or does not want to use BAT CLI to author complex test cases, user can use this UI to develop ping tests and advanced tests with assertions.

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 2]
```diff
@@ -55,6 +55,13 @@ Beneath the name of the monitor is an indicator that explains when the current r
 
 After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
 
+[IMPORTANT]
+====
+Tests silently fail when they take more than 120 seconds to run. As a workaround, you can split a long-running test into multiple tests. It is suggested to download the monitor, split the test across multiple monitors in the BAT CLI, run and time each test to be sure that it is under 120 seconds, and then upload the monitors.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/afm-create-monitor.adoc]: "It is suggested" > "MuleSoft suggests", or just the imperative > "Download the monitor, ..."

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 3]
```diff
@@ -55,6 +55,13 @@ Beneath the name of the monitor is an indicator that explains when the current r
...
+
+If you do not want to split a long-running test, you can download the monitor and run it from the BAT CLI.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/afm-create-monitor.adoc]: "you can download" > "download"

**Review by Cristian-Venticinque — APPROVED** [review]

File: modules/ROOT/pages/afm-create-monitor.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/afm-create-monitor.adoc:6]: @harshakirank , I'm referring to the language that is documented here in the AFM docs: https://docs.mulesoft.com/api-functional-monitoring/bat-bdd-reference

**Pull request merged**